firmware/Core/Inc/telecommands/agenda_telecommands.h

```diff
@@ -0,0 +1,12 @@
+#ifndef __INCLUDE_GUARD__AGENDA_TELECOMMAND_H
+#define __INCLUDE_GUARD__AGENDA_TELECOMMAND_H
+
+#include "telecommands/telecommand_types.h"
```

Comment:
Should this go in the `agenda_telecommand_defs.h` file?

Reply:
Migrated code to `agenda_telecommand_defs.h/c` files